### ISSUE TITLE
Don't break lines after opening quote or before ellipsis or closing quote

### ISSUE THREAD

**Issue by josteinaj** [nlbdev/pipeline#5]
_From @bertfrees on March 2, 2016 13:11_

For example, like this:

```
Lisa gikk langt til
sk…

«Lisa gikk til til gikk
til …»

Dette tror faktisk jeg 
også …

«Nå er det nok forfo i dag,»
sa hun.
```

and not like this:

```
Lisa gikk langt til sk
…

«Lisa gikk til til gikk til
…»

Dette tror faktisk jeg også
…

«Nå er det nok forfo i dag,
» sa hun.
```

_Copied from original issue: nlbdev/pipeline-mod-nlb#23_

**Comment by josteinaj** [comment]
_From @bertfrees on March 2, 2016 13:23_

In the cases where there is no space between the letter and the ellipsis or quote it is probably working fine already. In the cases with a space (2nd and 3rd examples) it is probably not working. A possible fix is to replace those spaces with no-break spaces.

@KariRudjord is first going to test some more.

**Comment by KariRudjord** [comment]
Right, @josteinaj and @bertfrees it is not working when there is a space before the ellipse. A no-break space would be nice!

A quote shall always stand together with the word, and it works fine!!

**Comment by josteinaj** [comment]
Ok, thanks. We'll fix this during pre-processing later.

**Comment by josteinaj** [comment]
Added related issue to NLBPUB project here: nlbdev/nlbpub#6

**Comment by KariRudjord** [comment]
We need to put in a unicode hard space between the word and the ellipse (...) so that we avoid behaviour as this:
![image](https://user-images.githubusercontent.com/12409918/46725075-af5dbd80-cc7b-11e8-96e5-92be33d6cc47.png)
![image](https://user-images.githubusercontent.com/12409918/46725161-e3d17980-cc7b-11e8-89ad-13b52e986d0c.png)

**Comment by KariRudjord** [comment]
A preprosessing thing, not a part of the translation. Easy to fix. Bert will look into it.

**Comment by bertfrees** [comment]
Forgot to move this to "Ready for test". It was fixed in commit 0e91d95.
